js/apps/admin-ui/src/user/UserAttributes.tsx

```diff
@@ -36,10 +34,10 @@ export const UserAttributes = ({
         reset={() =>
           form.reset({
             ...form.getValues(),
-            attributes: toUserFormFields(user, isUserProfileEnabled).attributes,
+            attributes: toUserFormFields(user).attributes,
           })
         }
-        name={isUserProfileEnabled ? "unmanagedAttributes" : "attributes"}
+        name={"unmanagedAttributes"}
```

Comment:
This doesn't need to be an expression.
```suggestion
        name="unmanagedAttributes"
```